core/controllers/suggestion_test.py

```diff
@@ -945,13 +955,143 @@ def test_update_suggestion_updates_question_suggestion_content(self):
         self.assertEqual(
             new_question_state_data['content'][
                 'html'],
-            '<p>Updated question</p>')
+            (
+                '<p>Updated question</p>'
+                '<oppia-noninteractive-image filepath-with-value='
+                '"&quot;img.png&quot;" caption-with-value="&quot;&quot;" '
+                'alt-with-value="&quot;Image&quot;">'
+                '</oppia-noninteractive-image>'))
         self.assertEqual(
             new_question_state_data['interaction'][
                 'solution'],
             new_solution_dict)
         self.logout()
 
+    def test_update_question_suggestions_with_empty_image_data(self):
```

Comment:
Can you add the expected outcome of the test in the test name? e.g. `test_update_question_suggestion_with_empty_image_data_throws_some_error`. Here and below.

Also, the test mentions "empty image data", but the test refers to large images?